utils/session.c

```diff
@@ -695,7 +695,7 @@ struct uftrace_symbol * task_find_sym(struct uftrace_session_link *sessions,
 }
 
 /**
- * task_find_sym - find a symbol that matches to @addr
+ * task_find_sym_addr - find a symbol that matches to @addr
```

Comment:
As @namhyung mentioned, `tracee` is intentional, but we can leave this change. So the title of the commit is better to be changed.

Reply:
@honggyukim I changed my commit. I appreciate it.